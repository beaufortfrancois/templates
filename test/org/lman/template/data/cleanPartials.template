Inline partials should get squashed down: {{+partials.cleanPartials_p1}}.

The same partial on its own line shouldn't get squashed down:
{{+partials.cleanPartials_p1}}
(and shouldn't add any new lines before or after).

More funkiness when they're adjacent:
{{+partials.cleanPartials_p1}}
{{+partials.cleanPartials_p1}}
{{+partials.cleanPartials_p1}}
(woo).

Partials should be indented on the level that they're included:
  {{+partials.cleanPartials_p1}}
(still no newlines).

Even more funk:
  {{+partials.cleanPartials_p1}}
    {{+partials.cleanPartials_p1}}
  {{+partials.cleanPartials_p1}}
    {{+partials.cleanPartials_p1}}
(woo).

These should all be single lines; too hard to do it otherwise.
This is more consistent too.
{{?list}}{{+partials.cleanPartials_p1}}{{/}}
  {{?list}}{{+partials.cleanPartials_p1}}{{/}}

{{?list}}a {{+partials.cleanPartials_p1}}{{/}}
  {{?list}}a {{+partials.cleanPartials_p1}}{{/}}

{{#list}}{{+partials.cleanPartials_p1}}{{/}}
  {{#list}}{{+partials.cleanPartials_p1}}{{/}}

{{#list}}a {{+partials.cleanPartials_p1}}{{/}}
  {{#list}}a {{+partials.cleanPartials_p1}}{{/}}

{{?list}}{{#list}}{{+partials.cleanPartials_p1}}{{/}}{{/}}
  {{?list}}{{#list}}{{+partials.cleanPartials_p1}}{{/}}{{/}}
