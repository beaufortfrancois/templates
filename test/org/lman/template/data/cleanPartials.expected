Inline partials should get squashed down: One  TwoThree.

The same partial on its own line shouldn't get squashed down:
One
  Two
Three
(and shouldn't add any new lines before or after).

More funkiness when they're adjacent:
One
  Two
Three
One
  Two
Three
One
  Two
Three
(woo).

Partials should be indented on the level that they're included:
  One
    Two
  Three
(still no newlines).

Even more funk:
  One
    Two
  Three
    One
      Two
    Three
  One
    Two
  Three
    One
      Two
    Three
(woo).

These single-line sections should still render as blocks.
One
  Two
Three
  One
    Two
  Three
  
a One  TwoThree
  a One  TwoThree

One
  Two
Three
One
  Two
Three
One
  Two
Three
  One
    Two
  Three
  One
    Two
  Three
  One
    Two
  Three
  
a One  TwoThreea One  TwoThreea One  TwoThree
  a One  TwoThreea One  TwoThreea One  TwoThree

One
  Two
Three
One
  Two
Three
One
  Two
Three
  One
    Two
  Three
  One
    Two
  Three
  One
    Two
  Three
  note: bug here, shouldn't be indented.
